scripts/linux/electron-installer-appimage.sh

```diff
@@ -105,7 +105,6 @@ fi
 if [ -z "$TMPDIR" ]; then
   TMPDIR=$(mktemp -d)
```

Comment:
Ah! I'd missed the bit where you'd set `TMPDIR=` in the Makefile rule :)
IMHO it'd still be worth renaming this variable from `TMPDIR` to `TEMPORARY_DIRECTORY` though (both in this script and in the Makefile) because it probably makes sense to do things in a subdirectory of `$TMPDIR` (if it's set in the environment) rather than in`$TMPDIR` itself, and "a subdirectory of `$TMPDIR` (if it's set in the environment)" is exactly what `mktemp -d` gives us.